Assets/Scripts/Controllers/HeaderController.cs

```diff
@@ -38,30 +62,92 @@ public class HeaderController : MonoBehaviour {
 				headerTitle.text = Config.CurrentlyLookingAtText;
 				headerValue.text = nodeController.HighlightedNode.Title;
 			} else {
-				headerTitle.text = "";
-				headerValue.text = "";
+				headerTitle.text = string.Empty;
+				headerValue.text = string.Empty;
 			}
 			if (nodeController.SelectedNode != null) {
 				if (nodeController.HighlightedNode != null && nodeController.HighlightedNode != nodeController.SelectedNode) {
 					headerTitle.text = Config.CurrentlyLookingAtText;
 					headerValue.text = nodeController.HighlightedNode.Title;
+					ShowOnlyConnectionsCount(connectionController.GetNodeNeighbours(nodeController.HighlightedNode).ToArray().Length);
 				} else {
 					headerTitle.text = Config.CurrentlySelectedText;
 					headerValue.text = nodeController.SelectedNode.Title;
+					ShowOnlyConnectionsCount(null);
+				}
+			} else {
+				connectionController.OnConnectionRangeChanged?.Invoke(0, 0, 0);
+				ShowOnlyConnectionsCount(connectionController.GetNodeNeighbours(nodeController.HighlightedNode).ToArray().Length);
+			}
+		}
+
+		private void ShowOnlyConnectionsCount(int? count) {
+			TextMesh headerConnectionsRangeText = HeaderObject.transform.GetChild(2).GetComponent<TextMesh>();
+			if(count != null) {
+				headerConnectionsRangeText.text = string.Format("{0} <color=white>{1}</color>", Config.CurrentConnectionRangeText, count);
+			} else {
+				if (currentCount <= connectionController.ConnectionDistribution.MaxVisibleNumber) {
+					headerConnectionsRangeText.text = string.Format("{0} <color=white>{1} {2}</color>", Config.CurrentConnectionRangeText, Config.AllConnectionRangeText, currentCount);
+				} else {
+					headerConnectionsRangeText.text = string.Format("{0} <color=white>[{1}-{2}/{3}]</color>", Config.CurrentConnectionRangeText, currentStart, currentEnd, currentCount);
+				}
+			}
```

Comment:
Split string assignment into 
- ```"{0} <color=white>"```
- appended conditions
- ```"</color>"```
to reduce line length